### ISSUE TITLE
Tune label percentile hyperparameter (25th vs 33rd vs other)

### ISSUE THREAD

**Issue by edwinsteele** [edwinsteele/fuel-price-signal#27]
## What
The two-condition BUY label uses a percentile gate on the past-90-day price distribution to catch high-price plateaus:

\`\`\`python
label = 1 (BUY) if:
    future_min >= today_price - threshold   # no better deal coming in next H days
    AND today_price <= Nth percentile of past 90 days  # price is cheap in absolute terms
\`\`\`

The percentile threshold N is currently set to 33rd as a starting point. It hasn't been tuned.

## Why revisit
- 33rd percentile → ~15 cheap days per 45-day cycle
- 25th percentile → ~11 cheap days per 45-day cycle
- The right value is whichever threshold best aligns with "genuinely good buying opportunity" validated against the backtest
- Changing N changes the label definition, which changes the training set, which changes the model — so this is a design decision not a free knob

## Acceptance criteria
- [ ] Run labels with N=25, 33, and at least one other value (e.g. 40)
- [ ] For each: inspect the positive rate and distribution of BUY days across the cycle
- [ ] Validate against backtest: which value produces the best simulated purchasing outcomes?
- [ ] Document the chosen value and rationale in `labels.py` docstring
- [ ] Consider whether forward horizon H (currently 7d) should also be revisited at the same time (14d was floated as an alternative to better handle plateaus)

**Comment by edwinsteele** [comment]
Imported into Beads (bd) as part of #278 — issue tracking for this project now lives there. See AGENTS.md § Beads.
